sig-data-science/charter.md

```diff
@@ -6,11 +6,11 @@ This charter adheres to the conventions described in the [Operate First Governan
 
 The Data Science SIG is responsible for all data science and machine learning related initiatives within the Operate First community. However, this SIG's primary focus is to serve the following 3 main objectives:
 
-* Use Operate First as the go-to platform for incubating Open Source cloud native data science projects and defining best practices for developing and operating such Open Source cloud native data science projects.
+* Use Operate First as the go-to platform for incubating Open Source cloud Native Data Science projects and defining best practices for developing and operating such Open Source Cloud Native Data Science projects.
 
 * Leverage the public and openly available operations data generated by the Operate First platform to evaluate and develop AIOps tools to support the operation and the operators of the platform.
 
-* Serve as ambassadors to the wider Open Source community through outreach initiatives (blogs, talks, etc.) and support for onboarding of new Open Source data science projects to the Operate First platform.
+* Serve as ambassadors to the wider Open Source community through outreach initiatives (blogs, talks, etc.) and support for onboarding of new Open Source Data Science projects to the Operate First platform.
```

Comment:
It's looking as if lower case "data science" is common in the industry:

https://towardsdatascience.com/ -- all lower case unless in a title.
https://en.wikipedia.org/wiki/Data_science

Unless we can find a compelling case to be made for capitalizing it, I think we should consistently do it lower case:  "data science"